### DISCUSSION TITLE
How can I dynamically modify the blend mode of my precompiled material?

### DISCUSSION THREAD

**Discussion by NewbyThebest** [google/filament#7452] [Q&A]
I have written a material as follows:

**Comment by NewbyThebest** [comment]
material {
    name : particle_material,
    shadingModel : unlit,
    blending : transparent,
    flipUV : false,
}

fragment {
}

**Comment by NewbyThebest** [comment]
How can I modify my blending in real-time based on runtime needs?

**Reply by mbalajee** [reply]
Did you try setting the parameter on the material instance, like
```kotlin
loadMaterial() // filamat file is loaded & material object is initialized
material.defaultInstance.setParameter("blendMode", BlendMode.TRANSLUCENT)
```

**Reply by PsycoTodd** [reply]
What do you want to achieve? I feel there might be other ways rather than chaning blend mode.
